environment/docker/php/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM php:7.4.9-fpm-alpine
 
 RUN apk update && apk upgrade \ 
-    && apk add --no-cache php7-pear php7-dev gcc musl-dev make
+    && apk add --no-cache php7-pear php7-dev php7-intl gcc musl-dev make icu-dev
```

Comment:
I think we don't need to install the php7-intl, as you enabled intl extension a few lines below in the code.